code/obj/artifacts/artifact_objects/bombs.dm

```diff
@@ -208,11 +210,14 @@ ABSTRACT_TYPE(/datum/artifact/bomb)
 	associated_object = /obj/machinery/artifact/bomb/chemical
 	rarity_class = 2
 	explode_delay = 0
+	alarm_initial = null
+	alarm_during = null
+	alarm_final = null
 	react_xray = list(5,65,20,11,"HOLLOW")
 	validtypes = list("ancient","martian","eldritch","precursor")
 	validtriggers = list(/datum/artifact_trigger/force,/datum/artifact_trigger/heat,/datum/artifact_trigger/carbon_touch)
 	var/payload_type = 0 // 0 for smoke, 1 for foam, 2 for propellant, 3 for just dumping fluids
-	var/recharge_delay = 600
+	recharge_delay = 600
```

Comment:
may as well make this 10 MINUTES as well

Reply:
true